### PR TITLE
Allow running tests locally without Pkg.test

### PR DESCRIPTION
**Pull request by abelsiqueira**
Use `julia -E 'include("src/AmplNLReader.jl"); include("test/runtests.jl")'`.

### CODE REVIEW

File: src/ampl_model.jl [hunk 1]
```diff
@@ -28,7 +28,8 @@ type AmplModel <: AbstractNLPModel
 
   counters :: Counters       # Evaluation counters
 
-  function AmplModel(stub :: ASCIIString)
+  function AmplModel(stub)
+    stub = ascii(stub)
```
Comment (dpo) [inline src/ampl_model.jl]: What's the benefit of this?

Comment (abelsiqueira) [inline src/ampl_model.jl]: `joinpath(path, file)` returns UTF8String. This can be modified on the tests, but this seems more robust.

Comment (dpo) [inline src/ampl_model.jl]: What's their common abstract type? `String`?

Comment (abelsiqueira) [inline src/ampl_model.jl]: `AbstractString`. I'll change the `:: ASCIIString` to it and see if the `ccall` accepts.

**Comment by abelsiqueira** [comment]
Force-pushed with modification.

**Pull request merged**